.gitignore

```diff
@@ -4,7 +4,6 @@
 /node_modules
 /.pnp
 .pnp.js
-package-lock.json
```

Comment:
Side question: If we exclude `package-lock.json` from .gitignore, can we have a separate commit for this file in the future since it creates noise with the more significant changes from development?